libdnf/comps/group/sack.cpp

```diff
@@ -40,11 +39,11 @@ namespace libdnf::comps {
 
 
 GroupSackWeakPtr GroupSack::get_weak_ptr() {
-    return GroupSackWeakPtr(this, &p_impl->sack_guard);
+    return GroupSackWeakPtr(this, &sack_guard);
 }
 
 
-GroupSack::GroupSack(Comps & comps) : Sack(), comps{comps}, p_impl{new Impl()} {}
+GroupSack::GroupSack(Comps & comps) : Sack(), comps{comps} {}
 
 
 GroupSack::~GroupSack() {}
```

Comment:
I don't think you need the destructor anymore (as there's no longer a `p_impl`), you can delete it.

Reply:
When I try to delete it, there is en `error: invalid application of ‘sizeof’ to incomplete type ‘libdnf::comps::Group’`